src/Spotify.js

```diff
@@ -0,0 +1,37 @@
+import { URL } from 'url';
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.startsWith(item));
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    host === 'open.spotify.com' &&
+    includesSomeOfArray(pathname, [
```

Comment:
Also explicitly don't transform embed-links (for now). (see above)

```suggestion
    !includesSomeOfArray(pathname, ['embed', 'embed-podcast']) &&
    includesSomeOfArray(pathname, [
```

Reply:
Cool got it 👍